README.md

```diff
@@ -1364,9 +1364,19 @@ Valid options: `true` and `false`.
 
 Default value: `true`.
 
-##### `environment`
+##### `proxy_server`
 
-Environment variables for settings such as http_proxy, https_proxy, or ftp_proxy. These are passed through to the staging module and then to the underlying exec(s), so it follows the same format of the exec type [`environment`](https://docs.puppet.com/puppet/latest/reference/type.html#exec-attribute-environment).
+Specify a proxy server to use when downloading tomcat binaries. e.g.: `https://example.com:8080`
```

Comment:
* Change "Specify" to "Specifies"
* tomcat should be capitalized here (as the name of the Apache software).
* Change "e.g.:" to "For example,"
* If this URL value is a string that the user provides, it should be in single quotes rather than back ticks.